da/test/da_test.go

```diff
@@ -172,7 +176,7 @@ func startMockCelestiaNodeServer(t *testing.T) *cmock.Server {
 	return httpSrv
 }
 
-func doTestRetrieve(t *testing.T, dalc da.DataAvailabilityLayerClient) {
+func doTestRetrieve(ctx context.Context, t *testing.T, dalc da.DataAvailabilityLayerClient) {
```

Comment:
Actually, there is no point to pass context here - we can create `context.Background(()` inside this function (it makes more sense IMHO, as each test will have it's own context).